components/App.tsx

```diff
@@ -14,7 +14,7 @@ const games: string[] = [
 
 const languages: string[] = ["English", "German", "Spanish", "Japanese"];
 
-function App() {
+export default function App() {
```

Comment:
prefer src/components or not even components. src/Form/Form.tsx/Form.css

Reply:
done